tests/monster_test.fbs

```diff
@@ -6,6 +6,11 @@ namespace MyGame.Example2;
 
 table Monster {}  // Test having same name as below, but in different namespace.
 
+/// This will test a potential "ambiguous call to overloaded function" using the same data from different namespace.
+table MonsterStatus {
```

Comment:
This will add new generated code in all languages, which you'd have to add to the PR. Since the change was already in 2 cases, I'd leave these changes to the .fbs file out.